single-audit/single_audit/settings/production.py

```diff
@@ -0,0 +1,17 @@
+from .base import *
+
+SECRET_KEY = os.environ.get('SECRET_KEY')
+
+# Database
+# https://docs.djangoproject.com/en/2.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.environ.get('DATABASE_NAME'),
```

Comment:
We'll need to pull these from `VCAP_SERVICES` instead of `os.environ` once we're actually on ☁️.gov, but: we'll cross that bridge when we reach it!

Reply:
👍 👍